src/video/wayland/SDL_waylandwindow.c

```diff
@@ -600,7 +600,13 @@ decoration_frame_close(struct libdecor_frame *frame, void *user_data)
 static void
 decoration_frame_commit(struct libdecor_frame *frame, void *user_data)
 {
-    /* No-op */
+    SDL_WindowData *wind = user_data;
+
+    if (wind->sdlwindow->flags & SDL_WINDOW_OPENGL) {
+        SDL_GL_SwapWindow(wind->sdlwindow);
```

Comment:
This should probably be something like sending a `SDL_WINDOWEVENT_EXPOSED` (so that the window content gets redrawn), as I think "randomly" calling `eglSwapBuffers()` (which I assume is what this eventually results in) might not be a good idea, as that means potentially swapping to an empty or old buffer.

Reply:
Right, that was actually causing glitches when resizing while updating the title.